test/test.js

```diff
@@ -139,19 +139,27 @@ test("teardown", async() => {
   }
 });
 
-test("get contributors last activites for NodeSecure/scanner", async(tape) => {
-  const contributors = await github.getContributorsLastActivities("NodeSecure", "scanner");
+test("get contributor last activities", async(tape) => {
+  const [lastEvent] = await github.getContributorLastActivities({
+    owner: "NodeSecure",
+    repository: "scanner",
+    contributor: "fraxken"
+  });
 
-  tape.true(contributors.has("fraxken"));
+  tape.true(lastEvent.lastActivity);
```

Comment:
WDYT about having a more detailed assertion regarding what it is in `lastActivity`? not the values, but the properties or types? (or maybe you fear that octokit change their API too often)